blockstack-sdk/src/main/res/layout/activity_connect.xml

```diff
@@ -0,0 +1,186 @@
+                app:layout_constraintTop_toBottomOf="@id/connect_secret_key_desc_title" />
+
+            <ImageView
+                android:id="@+id/connect_tracking_desc_icon"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="4dp"
+                android:importantForAccessibility="no"
+                android:paddingLeft="16dp"
+                android:paddingRight="16dp"
+                android:src="@drawable/ic_track"
+                android:tint="?colorPrimary"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/connect_tracking_desc_title" />
+
+            <TextView
+                android:id="@+id/connect_tracking_desc_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="16dp"
+                android:gravity="top"
+                android:text="@string/connect_activity_tracking_desc"
+                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/connect_tracking_desc_icon"
+                app:layout_constraintTop_toBottomOf="@+id/tracking_divider" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+    </androidx.core.widget.NestedScrollView>
+
+
+    <com.google.android.material.button.MaterialButton
+        style="?blockstackMainButtonStyle"
+        android:id="@+id/connect_get_secret_key"
+        android:layout_height="50dp"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="32dp"
+        android:layout_width="match_parent"
+        android:text="@string/org_blockstack_login"
+        android:textColor="@android:color/white" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+    <com.google.android.material.button.MaterialButton
+        style="?blockstackBottomButtonStyle"
+        android:id="@+id/connect_sign_in"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_weight="1"
+        android:layout_marginStart="16dp"
+        android:layout_marginBottom="16dp"
+        android:text="@string/connect_dialog_restore" />
+
+    <com.google.android.material.button.MaterialButton
+        style="?blockstackBottomButtonStyle"
+        android:id="@+id/connect_how_it_works"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_weight="1"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="4dp"
```

Comment:
This margin should be simetric between the 2 buttons, otherwise one will be smaller than the other.